### PR TITLE
Study View: group comparison selection update

### PR DESCRIPTION
**Pull request by kalletlak**
Fix https://github.com/cBioPortal/cbioportal/issues/7186

### CODE REVIEW

File: src/pages/groupComparison/comparisonGroupManager/ComparisonGroupManager.tsx [hunk 1]
```diff
@@ -272,7 +272,7 @@ export default class ComparisonGroupManager extends React.Component<
                         this.props.store.updateComparisonGroupsFilter();
                     }}
                 >
-                    View
+                    Select
```
Comment (adamabeshouse) [inline src/pages/groupComparison/comparisonGroupManager/ComparisonGroupManager.tsx]: probably should also rename to `selectButton` and look elswhere for `view` to replace it with `select`

Comment (kalletlak) [reply src/pages/groupComparison/comparisonGroupManager/ComparisonGroupManager.tsx]: @jjgao should I rename it to `selectButton`?
@adamabeshouse I haven't found any `view` in group comparison related pages.

File: src/pages/groupComparison/comparisonGroupManager/ComparisonGroupManager.tsx [hunk 2]
```diff
@@ -272,7 +272,7 @@ export default class ComparisonGroupManager extends React.Component<
                         this.props.store.updateComparisonGroupsFilter();
                     }}
                 >
-                    View
+                    Filter
```
Comment (alisman) [inline src/pages/groupComparison/comparisonGroupManager/ComparisonGroupManager.tsx]: I thought it was "Select".   I can see "filter" as well, but maybe "Apply to filter" ???  @jjgao can you make an executive decision here?

Comment (alisman) [reply src/pages/groupComparison/comparisonGroupManager/ComparisonGroupManager.tsx]: And do change the comment above and the name of the getter accordingly

**Pull request merged**